src/main/java/org/minions/devfund/sergio/battleship_game/Ocean.java

```diff
@@ -0,0 +1,53 @@
+package org.minions.devfund.sergio.battleship_game;
+
+import java.util.Random;
+
+public class Ocean {
+
+    private int shots = 0;
+    private int hitCount = 0;
+    private int sunkShips = 0;
+
+    public Ocean() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1186)